CSS/style.css

```diff
@@ -0,0 +1,184 @@
+@import url('https://fonts.googleapis.com/css2?family=Clicker+Script&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+    margin: 0%;
+    padding: 0%;
+    background-color: #FFF;
+}
```

Comment:
- **[OPTIONAL]** This was not explicited by previous code reviewers but please 🙏🏽 add the correct background image to your page to be similar to the Figma template 👇🏽 

|Figma template | Your design |
|---|---|
|![image](https://user-images.githubusercontent.com/58642949/219681965-5343f1fb-f7de-4db0-b69f-416f5c3ad7f8.png) | ![image](https://user-images.githubusercontent.com/58642949/219682714-e073d2de-79d7-42cb-a6b7-41f5b793047c.png) |